recipes/amptk/meta.yaml

```diff
@@ -40,3 +40,7 @@ about:
   license_file: LICENSE.md
   summary: 'AMPtk: Amplicon tool kit for processing high throughput amplicon sequencing data.'
   license_family: BSD
+
```

Comment:
Thanks for adding the DOI. If you only add this you don't need to bump the build number. I'm wondering why you needed to change the checksum.

Reply:
source changed slightly due to a fixed typo, if bumping the version number is better I can do that as well.